nat-lab/bin/nordlynx

```diff
@@ -21,9 +21,26 @@ sleep 2
 ip addr s > /var/log/ipaddr.show
 nlx > /var/log/nlx.log
 /usr/bin/fakefm/fakefm -s testing123 2>&1 > /var/log/fakefm.log &
-/opt/nlx-radius/nlx-radius --log=stdout --verb=DEBUG 2>&1 > /var/log/nordlynx.log &
+/opt/nlx-radius/nlx-radius 2>&1 > /var/log/nordlynx.log &
```

Comment:
why debug log was removed?

Reply:
ah nvm, I see... :laughing:

Reply:
Those are no longer required with the new radius